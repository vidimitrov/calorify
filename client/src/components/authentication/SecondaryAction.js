import styled from 'styled-components';

const SecondaryAction = styled.div`
  margin: 0 0 20px;
`;

export default SecondaryAction;
