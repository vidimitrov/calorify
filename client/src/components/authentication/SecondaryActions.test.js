import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import SecondaryAction from './SecondaryAction';

it('Authentication/SecondaryAction component renders correctly', () => {
  const tree = renderer.create(<SecondaryAction />).toJSON();
  expect(tree).toMatchSnapshot();
});
