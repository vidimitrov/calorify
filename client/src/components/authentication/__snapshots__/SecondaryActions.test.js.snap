// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Authentication/SecondaryAction component renders correctly 1`] = `
.c0 {
  margin: 0 0 20px;
}

<div
  className="c0"
/>
`;
